examples/02_meta-analyses/12_plot_ibma_workflow.py

```diff
@@ -0,0 +1,114 @@
+"""
+
+.. _ibma_workflow:
+
+====================================================
+Run an image-based meta-analysis (IBMA) workflow
+====================================================
+
+NiMARE provides a plethora of tools for performing meta-analyses on neuroimaging data.
+Sometimes it's difficult to know where to start, especially if you're new to meta-analysis.
+This tutorial will walk you through using a IBMA workflow function which puts together
+the fundamental steps of a IBMA meta-analysis.
+"""
+import os
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+from nilearn.plotting import plot_stat_map
+
+from nimare.extract import download_nidm_pain
+
+###############################################################################
+# Download data
+# -----------------------------------------------------------------------------
+
+dset_dir = download_nidm_pain()
+
+###############################################################################
+# Load Dataset
+# -----------------------------------------------------------------------------
+import nibabel as nib
+from nilearn.image import resample_to_img
+
+from nimare.dataset import Dataset
+from nimare.transforms import ImageTransformer
+from nimare.utils import get_resource_path
+
+dset_file = os.path.join(get_resource_path(), "nidm_pain_dset.json")
+dset = Dataset(dset_file)
+dset.update_path(dset_dir)
+
+# Resample images
+imgs = dset.get({"z_maps": ("image", "z")})["z_maps"]
+mask_img = dset.masker.mask_img
+for img in imgs:
+    res_img = resample_to_img(nib.load(img), mask_img, clip=True, interpolation="linear")
+    res_img.to_filename(img)
```

Comment:
Is resampling not being handled here?

https://github.com/neurostuff/NiMARE/blob/e833c4bcb4a921ddea7b7f8c7c1aab28fec7dfa8/nimare/meta/ibma.py#L76-L88

Reply:
I think resampling is performed if `resample=True` in the IBMA estimator. Since we are initializing the estimator with the default parameters, where `resample=False`, that step is skipped.

Reply:
I think we should initialize the estimator with resample=true, it means we can't use simple strings and have to import the estimator, but I think that's better than what we are currently doing.